types/yup/index.d.ts

```diff
@@ -102,6 +98,8 @@ export interface MixedSchemaConstructor {
 
 // tslint:disable-next-line:no-empty-interface
 export interface MixedSchema<T = any> extends Schema<T> {
+    nullable(isNullable?: true): MixedSchema<T | null>;
+    nullable(isNullable: false): MixedSchema<Exclude<T, null>>;
```

Comment:
can `nullable` be called with anything but a literal? It doesn't work with an unknown boolean:

```ts
function wrapper(b: boolean, msx: MixedSchema) {
  return msx.nullable(b)
}
```

Gives an error "Argument of type 'boolean' is not assignable to 'false'."